<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiodeSelector</class>
 <widget class="QDialog" name="DiodeSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Double Diodes</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="singleDiodeRadioButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Single diode</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="commonAnodeRadioButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Common anode</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="commonCathodeRadioButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Common cathode</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="connectedInSeriesRadioButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Connected in series</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
